stingray/tests/test_spectroscopy.py

```diff
@@ -395,3 +391,47 @@ def test_compute_rms():
 
     with pytest.raises(ValueError):
         spec.compute_rms(cs, model, criteria="filter")
+
+
+def test_get_mean_phase_difference():
+    dt = 0.01
+    time = np.arange(0, 100, dt)
+    qpo_freq = 0.1
+    harmonic_freq = 2 * qpo_freq
+    counts_qpo = 100 * np.sin(2 * np.pi * qpo_freq * time - np.pi / 4)
+    counts_harmonic = 50 * np.sin(2 * np.pi * harmonic_freq * time - np.pi / 4)
```

Comment:
I'm a bit confused by the phases here - it looks like both are -pi/4. Should they not be -pi/4 for one, and +pi/4 for the other, if you want a difference of pi/2?